src/pop/entities/vbktx.cpp

```diff
@@ -62,6 +62,16 @@ uint256 VbkTx::getHash() const {
   return sha256(stream.data());
 }
 
+Coin VbkTx::calculateTxFee() const {
+  decltype(Coin::units) output{0};
+  for (const auto& o : outputs) {
+    output += o.coin.units;
+  }
+  VBK_ASSERT_MSG(sourceAmount.units >= output,
```

Comment:
I think this should be removed, because otherwise malitious actor could send bad tx and all nodes will fail on this assert

Reply:
alternatively you can add validation of fees

Reply:
hmmmm. Okay. I think we need to do fee validation (check veriblock rules) and leave this assert here then